app-ios/tutanota/Sources/Utils/UserPreferences.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+/// User settings provider
+///
+/// This is abstracted into an interface for better testability.
+protocol UserPreferencesProvider {
+  /// Get the settings object for the given key.
+  ///
+  /// Parameters:
+  /// - `forkey` key to get settings
+  func getObject(forKey: String) -> Any?
```

Comment:
I would try to find a solution that uses generics instead of disabling the type system. [See also this comment](https://github.com/tutao/tutanota/pull/6437/files#r1469197811).